jme3-core/src/main/java/com/jme3/renderer/Camera.java

```diff
@@ -580,6 +581,43 @@ public void setFrustumTop(float frustumTop) {
         onFrustumChange();
     }
 
+    /**
+     * Obtains field of view when the camera is in perspective mode.
+     *
+     * @return Frame of view angle along the Y in degrees, or -1 if the camera is in orthogonal mode.
+     */
+    public float getFov() {
+        return !this.parallelProjection ? currentFov : -1;
+    }
+
+    /**
+     * Sets the field of view when the camera is in perspective mode. Note that this method has no
+     * effect when the camera is in orthogonal mode.
+     *
+     * @param fovY   Frame of view angle along the Y in degrees.
+     */
+    public void setFov(float fovY) {
+        if (!this.parallelProjection && fovY > 0) {
```

Comment:
I think passing a non-positive argument should throw an exception.

Reply:
Makes sense. Should we throw an exception for trying to set the FOV on orthogonal cameras as well, or just let them silently fail?

Reply:
I lean toward throwing an exception. Silent failures lead to subtle bugs.